src/helpers/array.ts

```diff
@@ -1,5 +1,11 @@
 import { pullAt } from "lodash";
 
+export function notEmpty<TValue>(
```

Comment:
You can use lodash compact on the array instead.

Reply:
Oh gotcha. Okay, I'll fix it.